lib/messages.rb

```diff
@@ -0,0 +1,37 @@
+require 'pg'
+
+class Messages
+
+  attr_reader :id, :message, :name
+
+  def initialize(id:, message:, name:) #(id:, name:, message:)
+    @id = id
+    @message = message
+    @name = name
```

Comment:
Should a peep be responsible for knowing the authors name?
Perhaps that should be a responsibility of an Author, or User class?